pandas/core/strings/object_array.py

```diff
@@ -430,3 +430,6 @@ def _str_lstrip(self, to_strip=None):
 
     def _str_rstrip(self, to_strip=None):
         return self._str_map(lambda x: x.rstrip(to_strip))
+
+    def _str_removeprefix(self, prefix=None):
+        return self._str_map(lambda x: x.removeprefix(prefix))
```

Comment:
this only exists in 3.9 right? is there an impl that can work for other versions as well

Reply:
This is the spec from [PEP 616](https://www.python.org/dev/peps/pep-0616/#specification):

``` python
def removeprefix(self: str, prefix: str, /) -> str:
    if self.startswith(prefix):
        return self[len(prefix):]
    else:
        return self[:]

def removesuffix(self: str, suffix: str, /) -> str:
    # suffix='' should not call self[:-0].
    if suffix and self.endswith(suffix):
        return self[:-len(suffix)]
    else:
        return self[:]
```